packages/medusa/src/api/middlewares/transform-includes-options.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response } from "express"
+import { FindConfig } from "../../types/common"
+import { Order } from "../../models"
+
+/**
+ * Retrieve the includes options from the select prop of retrieveConfig.
+ * If the include option is present then assigned it to includes on req
+ * @param allowedIncludesFields The list of fields that can be passed and assign to req.includes
+ */
+export function transformIncludesOptions(allowedIncludesFields: string[]) {
```

Comment:
thought: Wouldn't it be better to default to an empty array here instead of having to remember to do: `transformIncludesOptions([])`?